### ISSUE TITLE
Sluggish dragging

### ISSUE THREAD

**Issue by pierot** [deanmcpherson/react-native-sortable-listview#39]
We implemented the package successfully in a project but in the layout setup we now use it behaves badly, very sluggish. See https://youtu.be/TKfnQ_vEnDY as reference.

The layout is currently build up like this:

``` jsx
<View style={{flex: 1, backgroundColor: Colors.white}}>
  <Router>
    <Scene>
      <Scene>
        <Scene>
          <SortableListView 
            renderRow={row => <Row /* TouchableHighlight as seen in the examples */ />} 
          />
        </Scene>
      </Scene>
    </Scene>
  </Router>
</View>
```

SortableListView has no flex style, but a fixed height.

**Comment by huchenme** [comment]
+1, facing same issue

**Comment by AhsanWani** [comment]
+1, same here

**Comment by deanmcpherson** [comment]
Hey @pierot, sorry for the slow reply. Is there any code you can share to reproduce? Also, what version of React Native are you on?

**Comment by codesinghanoop** [comment]
+1

**Comment by mamidala** [comment]
+1, am too facing the issue, and in android it is very sluggish.

**Comment by Jarred-Sumner** [comment]
+1

**Comment by chetstone** [comment]
In my testing, PR #61 seems to help this problem quite a bit

**Comment by chetstone** [comment]
Should be fixed by f5c2ae0cc55f3d8c332ee3ba5b8d3eb568520739. Please reopen if not.

**Comment by Jarred-Sumner** [comment]
After upgrading this package and React Native to the latest version, I'm no longer able to build the project :(

```
➜  alarmclockv2 git:(master) ✗ react-native start
Scanning 647 folders for symlinks in /Users/jarred/Code/alarmclockv2/node_modules (14ms)
 ┌────────────────────────────────────────────────────────────────────────────┐
 │  Running packager on port 8081.                                            │
 │                                                                            │
 │  Keep this packager running while developing on any JS projects. Feel      │
 │  free to close this tab and run your own packager instance if you          │
 │  prefer.                                                                   │
 │                                                                            │
 │  https://github.com/facebook/react-native                                  │
 │                                                                            │
 └────────────────────────────────────────────────────────────────────────────┘
Looking for JS files in
   /Users/jarred/Code/alarmclockv2

Loading dependency graph...
React packager ready.

Failed to build DependencyGraph: @providesModule naming collision:
  Duplicate module name: jsc-heap-capture
  Paths: /Users/jarred/Code/alarmclockv2/node_modules/react-native-sortable-listview/node_modules/react-native/local-cli/server/middleware/heapCapture/package.json collides with /Users/jarred/Code/alarmclockv2/node_modules/react-native/local-cli/server/middleware/heapCapture/package.json

This error is caused by a @providesModule declaration with the same name across two different files.
Error: @providesModule naming collision:
  Duplicate module name: jsc-heap-capture
  Paths: /Users/jarred/Code/alarmclockv2/node_modules/react-native-sortable-listview/node_modules/react-native/local-cli/server/middleware/heapCapture/package.json collides with /Users/jarred/Code/alarmclockv2/node_modules/react-native/local-cli/server/middleware/heapCapture/package.json

This error is caused by a @providesModule declaration with the same name across two different files.
    at HasteMap._updateHasteMap (/Users/jarred/Code/alarmclockv2/node_modules/react-native/packager/src/node-haste/DependencyGraph/HasteMap.js:158:13)
    at p.getName.then.name (/Users/jarred/Code/alarmclockv2/node_modules/react-native/packager/src/node-haste/DependencyGraph/HasteMap.js:133:31)
```
